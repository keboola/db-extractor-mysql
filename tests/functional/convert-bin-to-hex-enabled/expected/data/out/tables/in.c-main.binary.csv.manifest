{
    "destination": "in.c-main.binary",
    "incremental": false,
    "metadata": [
        {
            "key": "KBC.name",
            "value": "binary_conversions"
        },
        {
            "key": "KBC.sanitizedName",
            "value": "binary_conversions"
        },
        {
            "key": "KBC.schema",
            "value": "test"
        },
        {
            "key": "KBC.type",
            "value": "BASE TABLE"
        },
        {
            "key": "KBC.rowCount",
            "value": 3
        }
    ],
    "columns": [
        "bin1",
        "bin2"
    ],
    "column_metadata": {
        "bin1": [
            {
                "key": "KBC.datatype.nullable",
                "value": true
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "STRING"
            },
            {
                "key": "KBC.datatype.default",
                "value": ""
            },
            {
                "key": "KBC.sourceName",
                "value": "bin1"
            },
            {
                "key": "KBC.sanitizedName",
                "value": "bin1"
            },
            {
                "key": "KBC.uniqueKey",
                "value": false
            },
            {
                "key": "KBC.ordinalPosition",
                "value": 1
            }
        ],
        "bin2": [
            {
                "key": "KBC.datatype.nullable",
                "value": true
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "STRING"
            },
            {
                "key": "KBC.datatype.default",
                "value": ""
            },
            {
                "key": "KBC.sourceName",
                "value": "bin2"
            },
            {
                "key": "KBC.sanitizedName",
                "value": "bin2"
            },
            {
                "key": "KBC.uniqueKey",
                "value": false
            },
            {
                "key": "KBC.ordinalPosition",
                "value": 2
            }
        ]
    }
}