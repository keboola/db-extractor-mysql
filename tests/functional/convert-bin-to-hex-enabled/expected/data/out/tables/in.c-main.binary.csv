"3f1256","fbd300"
"aa4234","f12c40"
"ffa500","f0a3dd"
